app/controllers/studentship/course_enrollments_controller.rb

```diff
@@ -34,8 +34,11 @@ def destroy
 
     def save
       if @service.course_enrollments.any?
```

Comment:
iç içe `if-else` block'larının yerine guard-clause kullanılabilir. popüler projelerde ve rails dokümanında gördüğüm kadarıyla `return` ve `redirect`in birlikte kullanımında bir sakınca yok